src/main/java/marquez/MarquezContext.java

```diff
@@ -39,46 +47,55 @@
 import marquez.service.models.Tag;
 import org.jdbi.v3.core.Jdbi;
 
+@Getter
 public final class MarquezContext {
-  @Getter private final NamespaceDao namespaceDao;
```

Comment:
For readability and consistency, mind if we keep instances variables annotated with `@Getter`? which is the approach taken for all of our [service models](https://github.com/MarquezProject/marquez/tree/main/src/main/java/marquez/service/models).